src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -575,8 +575,12 @@ def launch_fluent(
                     all_py_journal_names = ""
                     name_list = []
                     for journals in topy:
-                        journal_name = journals.rsplit(".")[0]
-                        journal_ext = journals.rsplit(".")[1]
+                        if os.path.exists(journals):
+                            journal_name = journals.rsplit("\\")[-1].split(".")[0]
```

Comment:
@hpohekar 

The "\\\\" seems like a WIndows assumption. Shall we use something designed for the job that works with any os, e.g. os.path.split()? 

Also these two lines are unnecessarily duplicated - the rhs of a single statement can just be `name, ext = `.

Also there's no need for the if/else here. It works just the same.

Reply:
Ok. We will update this as well.

Reply:
(I was still editing the comment - please check all points and let me know if anything is unclear)

Reply:
> (I was still editing the comment - please check all points and let me know if anything is unclear)

Yes, sure. Thank you.

Reply:
> [Path(<path>).stem](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem) can also be used here. We probably don't need to extract the extension here.

I see. Thank you.

Reply:
Done. Thank you.